apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.StressTests/Program.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using MySql.Data.MySqlClient;
+
+namespace Google.Cloud.Spanner.Data.StressTests
+{
+    internal static class Program
+    {
+        private const bool PreWarmSpanner = true;
+        private const bool PreWarmMySql = true;
+
+        private const int TargetQps = 100;
+        private static readonly TimeSpan s_testDurationMs = TimeSpan.FromSeconds(120);
+
+        public const string SpannerConnectionString =
+            "Data Source=projects/spanneref/instances/myspanner/databases/books";
+
+        public const string MySqlConnectionString =
+            "Server=104.198.57.196;Database=books;Uid=root;Pwd=;Max Pool Size=400";
+
+        // ReSharper disable once UnusedParameter.Local
+        private static void Main(string[] args)
+        {
+            //Uncomment the following lines to test session limiting.
+            //            SpannerOptions.Instance.MaximumActiveSessions = 5;
+            //            SpannerOptions.Instance.MaximumPooledSessions = 5;
```

Comment:
I always run these tests from visual studio and never from the command line.
I think seeing code comments is much more discoverable than say a build config or command line flag.  At some point capped session pool behavior will have its own unit or integration test.